article/_posts/2020-08-14-IsolatingSourcesOfDisentanglementInVAEs.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: review
+title: "Isolating Sources of Disentanglement in VAEs"
+tags: deep-learning autoencoder VAE
+author: "Nathan Painchaud"
+cite:
+    authors: "Ricky T. Q. Chen, Xuechen Li, Roger Grosse, David Duvenaud"
+    title:   "Isolating Sources of Disentanglement in VAEs"
+    venue:   "NeurIPS 2018"
+pdf: "https://arxiv.org/pdf/1802.04942.pdf"
+---
+
+
+# Highlights
+- Decomposition of the variational lower bound that explains the success of $$\beta$$-VAE in learning disentangled
+representations
+- Simple method based on weighted minibatches to estimate the aggregate posterior without additional hyperparameters
+(the aggregate posterior is required to weight specific terms in the decomposition of the ELBO)
+- New $$\beta$$-TCVAE (Total Correlation Variational Autoencoder), which can be used as a plug-in replacement for the
+$$\beta$$-VAE (and VAE for that matter)
+- New information-theoretic disentanglement metric, which is classifier-free and generalizable
+
+
+# Introduction
+The paper was written independently from a similar paper by Kim et Mnih[^1], and in the same vein propose a tweak on
+the $$\beta$$-VAE[^2] to further disentangle the learned representation without sacrificing the reconstruction,
+although the modifications proposed by both papers differ.
+
+Lets recall the objective of the $$\beta$$-VAE:
+
+$$
+\tag{1} \mathcal{L}_{\beta} = \frac{1}{N} \sum_{n=1}^{N} (\mathbb{E}_{q} [\log p(x_{n}|z)] - \beta KL(q(z|x_{n})~\|~p(z)))
+$$
+
+where $$\beta \gt 1$$ to encourage disentanglement. This is a more general formulation of the VAE, since we can obtain
+the original VAE objective by setting $$\beta = 1$$ in eq. 1.
```

Comment:
_eq. 1_ -> Equations are not numbered when introducing them using LaTeX, right? At least not the way we usually write them; I ignore if the LaTeX `equation` environment can be used.

Reply:
Good question! That's one of KaTeX's functionalities that I discovered. You can number your equation by adding `tag{eq_tag}` before it (inside the `$$` markers), where `eq_tag` is the tag you want your equation to display.

[Here is an example](https://katex.org/docs/supported.html#annotation) from KaTeX's documentation.

Reply:
:+1: Sorry, my bad. I did not pay attention to the `tag{}` that was inside the math environment. Thanks for the explanation.

Reply:
No need to be sorry. I only just learned about it recently myself :wink: